frontend/src/__tests__/cypress/cypress/tests/mocked/modelServing/runtime/servingRuntimeList.cy.ts

```diff
@@ -1819,6 +1822,142 @@ describe('Serving Runtime List', () => {
       );
       kserveModal.findSubmitButton().should('be.enabled');
     });
+
+    it('Deploy OCI Model and check paste functionality', () => {
+      initIntercepts({
+        disableModelMeshConfig: false,
+        disableKServeConfig: false,
+        disableServingRuntimeParams: false,
+        servingRuntimes: [],
+        requiredCapabilities: [StackCapability.SERVICE_MESH, StackCapability.SERVICE_MESH_AUTHZ],
+        projectEnableModelMesh: false,
+      });
+
+      cy.interceptK8sList(
+        SecretModel,
+        mockK8sResourceList([
+          mockCustomSecretK8sResource({
+            namespace: 'test-project',
+            name: 'test-secret',
+            annotations: {
```

Comment:
Technically this should have the `type: 'kubernetes.io/dockerconfigjson'` probably?
```suggestion
          mockCustomSecretK8sResource({
            type: 'kubernetes.io/dockerconfigjson'
            namespace: 'test-project',
            name: 'test-secret',
            annotations: {
```